Lib/test/test_msvcrt.py

```diff
@@ -62,33 +61,45 @@ def test_get_osfhandle(self):
 
 class TestConsoleIO(unittest.TestCase):
     def test_kbhit(self):
-        h = msvcrt.get_osfhandle(sys.stdin.fileno())
-        flush_console_input_buffer(h)
-        self.assertEqual(msvcrt.kbhit(), 0)
+        # Run test in a seprated process to avoid stdin conflicts.
+        # See: gh-110147
+        cmd = [sys.executable, '-c',
+               'import msvcrt;'
+               'assert msvcrt.kbhit() == 0;'
+               ]
+        subprocess.run(cmd, check=True, capture_output=True)
 
     def test_getch(self):
         msvcrt.ungetch(b'c')
         self.assertEqual(msvcrt.getch(), b'c')
 
     def test_getwch(self):
-        with open('CONIN$', 'rb', buffering=0) as stdin:
-            h = msvcrt.get_osfhandle(stdin.fileno())
-            flush_console_input_buffer(h)
-
-            write_input(stdin, c_encoded)
-            self.assertEqual(msvcrt.getwch(), c)
+        # Run test in a seprated process to avoid stdin conflicts.
+        # See: gh-110147
+        cmd = [sys.executable, '-c',
```

Comment:
Can you use textwrap.dedent() to make the code more readable?

```py
code = textwrap.dedent("""
   ... your code ...
   ... on multiple lines ...
""")
```

Reply:
Done.